modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/CommandArgIterator.java

```diff
@@ -45,9 +49,14 @@ public class CommandArgIterator {
      * @param argsIt Raw argument iterator.
      * @param commonArgumentsAndHighLevelCommandSet All known subcomands.
      */
-    public CommandArgIterator(Iterator<String> argsIt, Set<String> commonArgumentsAndHighLevelCommandSet) {
+    public CommandArgIterator(
+        Iterator<String> argsIt,
+        Set<String> commonArgumentsAndHighLevelCommandSet,
+        Map<String, Command<?>> cmds
```

Comment:
We need to describe the new parameter in javadoc.

Reply:
Done